js/stores/appStore.js

```diff
@@ -485,6 +487,32 @@ const handleAppAction = (action) => {
     case appConstants.APP_DATA_URL_COPIED:
       nativeImage.copyDataURL(action.dataURL, action.html, action.text)
       break
+    case appConstants.APP_APPLY_SITE_RECORDS:
```

Comment:
or maybe the comment should go on the action. Either way is fine, just want to capture it in the code